module1/homework/grayscaleImages/compression.hpp

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include <array>
+#include <cstdint> // uint8_t
+#include <vector>
+
+constexpr size_t width = 32;
+constexpr size_t height = 32;
+constexpr unsigned char min_printable = 33;
+
+std::vector<std::pair<uint8_t, uint8_t>> compressGrayscale(
+    const std::array<std::array<uint8_t, width>, height>&);
+std::array<std::array<uint8_t, width>, height> decompressGrayscale(
```

Comment:
Have youn considered using alias for longer names like std::array<std::array<uint8_t, width>, height>?